pkgs/development/python-modules/pytest-tornasync/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, buildPythonPackage, fetchFromGitHub
+, pytest, tornado }:
+
+buildPythonPackage rec {
+  pname = "pytest-tornasync";
+  version = "0.6.0.post2";
```

```suggestion
  version = "0.6.0.post2";
  disabled = isPy27;
```
builder for '/nix/store/1qj7cyp9723msq2vl1lzq5wgwgm926ar-python2.7-pytest-tornasync-0.6.0.post2.drv' failed with exit code 1; last 10 log lines:
  unpacking sources
  unpacking source archive /nix/store/lzzb7w2hxgaji81yf3426308lca56awh-source
  source root is source
  setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/tox.ini
  patching sources
  configuring
  no configure script, doing nothing
  building
  Executing setuptoolsBuildPhase
  pytest-tornasync requires Python 3.5 or newer
```

---

Hah, weird. I tought I already pushed that back when I remarked about it in https://github.com/NixOS/nixpkgs/pull/94568#issuecomment-668057659. :tired_face:

---

Done, thanks!